corehq/apps/app_manager/xpath_validator/js/xpathValidator.js

```diff
@@ -0,0 +1,11 @@
+global.SchemeNumber = require('js-xpath/lib/schemeNumber').SchemeNumber;
```

Comment:
I think this might actually be able to be solved in js-xpath now.

https://github.com/dimagi/js-xpath/blob/master/models.js#L103 is the only place this is used. Since it now uses browserify to build xpath.js you should be able to require schemenumber in `models.js` and use from there.

Could do something like this:https://github.com/dimagi/js-xpath/blob/7cbc5f765c922c583a3c9650a13948c17b3c4db3/lib/schemeNumber.js#L16